lib/spack/spack/cmd/tutorial.py

```diff
@@ -24,7 +24,7 @@
 
 
 # tutorial configuration parameters
-tutorial_branch = "releases/v0.18"
+tutorial_branch = "releases/v0.19"
```

Comment:
Should we just update this to:
```suggestion
tutorial_branch = "releases/v%s" % ".".join(spack.spack_version_info[:2])
```

and skip #33860?

Reply:
That is a great idea